utils/connect.go

```diff
@@ -10,34 +10,43 @@ import (
 	"github.com/BoltzExchange/boltz-client/logger"
 )
 
-func ConnectBoltz(lightning lightning.LightningNode, boltz *boltz.Boltz) (string, error) {
-	nodes, err := boltz.GetNodes()
+func connect(node lightning.LightningNode, peer *boltz.NodeInfo) error {
+	if len(peer.Uris) == 0 {
+		return fmt.Errorf("no uris found for peer: %s", peer.PublicKey)
+	}
+	uri := peer.Uris[0]
```

Comment:
We should prefer clearnet URIs. The backend orders the URIs that way already, but just to the be sure, it should be done here too